docs/source/concepts.rst

```diff
@@ -43,7 +43,7 @@ the run:
 - *epoch is started/completed*
 - *batch iteration is started/completed*
 
-Complete list of events can be found `here <https://github.com/pytorch/ignite/blob/master/ignite/engine/engine.py#L13>`_.
+Complete list of events can be found at :class:`ignite.engine.Events`.
```

Comment:
we should keep the link no?

Reply:
I think this is bad to keep track of the line in the repository as we could add some more code in the file and this link will become obsolete. 
This rst notation :class:`ignite.engine.Events` creates a link to the doc like https://pytorch.org/ignite/engine.html#ignite.engine.Events

Reply:
ah ok that's  perfect then